lib/publickey.js

```diff
@@ -384,6 +385,15 @@ PublicKey.prototype.toBuffer = PublicKey.prototype.toDER = function() {
   }
 };
 
+/*
+ * Will return a sha256 + ripemd160 hash of the serialized public key
+ * @see https://github.com/bitcoin/bitcoin/blob/master/src/pubkey.h#L141
+ * @returns {Buffer}
+ */
+PublicKey.prototype.getID = function getID() {
```

Comment:
Can we rename this to `fingerprint`, `hash`, or `hash160` and/or make this start with `_`? I feel strange about "the ID of a public key"